pkg/adaptation/api.go

```diff
@@ -78,6 +78,7 @@ type (
 	LinuxMemory              = api.LinuxMemory
 	LinuxDevice              = api.LinuxDevice
 	LinuxDeviceCgroup        = api.LinuxDeviceCgroup
+	CDIDevice                = api.CDIDevice
```

Comment:
Not that's it's too important, but should we consider sorting this block?

Reply:
Yeah, not sure what would be the best for this. Originally I tried to have this is an sort of 'logical hierarchical order', but it's probably not really that now. Lexical sorting would clearly be one well defined order. Anyway, I'd not reshuffle it now in this PR, as that would be an unrelated change.

Reply:
Sure, let's revisit this in a follow-up.